common/archiver/filestore/queryParser.go

```diff
@@ -251,17 +251,17 @@ func convertToTime(timeStr string) (time.Time, error) {
 func convertStatusStr(statusStr string) (enumspb.WorkflowExecutionStatus, error) {
 	statusStr = strings.ToLower(strings.TrimSpace(statusStr))
 	switch statusStr {
-	case "completed", strconv.Itoa(int(enumspb.WORKFLOW_EXECUTION_STATUS_COMPLETED)):
+	case "completed", convert.Int32ToString(int32(enumspb.WORKFLOW_EXECUTION_STATUS_COMPLETED)):
```

Comment:
These are not shardIDs. Actually, can you change it to:
```suggestion
	case "completed", enumspb.WORKFLOW_EXECUTION_STATUS_COMPLETED.String():
```

Reply:
actually the intention here is to get the integer value for query parsing, string function will return the actual enum name, like `Completed`